ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -1289,17 +1289,18 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-azure-ipi-ovn
-- as: azure-ipi-ovn-hypershift-guest-f7
-  cron: 0 22 6,13,20,27 * *
+- as: azure-ipi-ovn-hypershift-etcd-encryption-guest-f7
```

Comment:
I want to highlight that, the created vault key will be kepted for a long time on azure side even we delete the whole resource group, cc @jinyunma if I am wrong, correct me. So think about if the `f7` frequency is proper here.

Reply:
correct, it is just soft deleted when we delete it or the whole resource group, and it will be deleted permanently after 90 days. High frequency might get issue of "keyvault xxx is already in use", and unable to create new keyvault.
Similar issue happened before in CI, see https://github.com/openshift/release/pull/48303 for details.

Reply:
Updated.

Reply:
@jinyunma btw, I saw several des ci jobs are selected for sippy, which are set with a high frequency, when you have time, can you help file a PR to lower down a bit for those jobs, maybe f14 is a good choice.